cosrlib/signals/commonsearch_host_pagerank.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from . import BaseSignal
+
+
+class Signal(BaseSignal):
+    """ Ranking signal based on the Harmonic Centrality domain ranking
+        from the WebDataCommons project.
+    """
+
+    def get_value(self, document, url_metadata):
+
+        rank = url_metadata["domain"].commonsearch_host_pagerank
+        if not rank:
+            return None
+
+        return float(rank)
```

Comment:
The value should be normalized between 0 and 1. (The maximum PageRank value is the one of the first line of the file, and the minimum is 0.15)